### PR TITLE
Add description for geoindex

### PR DESCRIPTION
**Pull request by yupeng9**

### CODE REVIEW

File: basics/indexing/geospatial-support.md [hunk 1]
```diff
@@ -70,3 +70,75 @@ Following geospatial functions are available out of the box in Pinot-
...
+    ]
+  },
+}
+```
+{% endcode %}
+
+So the query below will use the geoindex to filter the Starbucks stores within 5km of the given point in the bay area.
+
+```sql
+SELECT address, ST_DISTANCE(location_st_point, ST_Point(37, -122,1))
```
Comment (Jackie-Jiang) [inline basics/indexing/geospatial-support.md:124]: longitude before latitude, also space
```suggestion
SELECT address, ST_DISTANCE(location_st_point, ST_Point(-122, 37, 1))
```

File: basics/indexing/geospatial-support.md [hunk 2]
```diff
@@ -70,3 +70,75 @@ Following geospatial functions are available out of the box in Pinot-
...
+FROM starbucksStores
+WHERE ST_DISTANCE(location_st_point, ST_Point(37, -122, 1)) < 5000
```
Comment (Jackie-Jiang) [inline basics/indexing/geospatial-support.md:126]: Same here

**Review by Jackie-Jiang — APPROVED** [review]
LGTM otherwise

File: basics/indexing/geospatial-support.md (hunk 1 above)
Comment (yupeng9) [reply basics/indexing/geospatial-support.md:124]: Good catch. Forgot to swap it back.

**Pull request merged**